### PR TITLE
Update hosting_providers.json to include DreamHost

### PR DESCRIPTION
**Pull request by tylampella**

### CODE REVIEW

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -38,6 +38,16 @@
     "plan": "",
     "reviewed": "2019.5.31",
     "note": ""
+  },
+    {
```
Comment (ohemorange) [inline _data/hosting_providers.json]:
```suggestion
  {
```

File: _data/hosting_providers.json [hunk 2]
```diff
@@ -38,6 +38,16 @@
...
+    "name": "DreamHost",
+    "link": "https://www.dreamhost.com/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "https://www.dreamhost.com/hosting/ssl-tls-certificates/",
+    "plan": "",
+    "reviewed": "2019.5.31",
```
Comment (ohemorange) [inline _data/hosting_providers.json]:
```suggestion
    "reviewed": "2020.1.22",
```

**Comment by tylampella** [comment]
> Thanks for the addition, make a couple of small tweaks in the suggestions.
> 
> Just to confirm, when users are setting up a new server, there's no additional buttons they have to click or anything to get a certificate?

ah..... we have some that are automatic and some where they have click one button. All very easy and intuitive.

**Comment by ohemorange** [comment]
> ah..... we have some that are automatic and some where they have to click one button. All very easy and intuitive.

Awesome, glad to hear that! In that case, could you:
1. add a few words to the "note" field to say which products are which, and
2. if you have it, a link to information about which button the user has to click in the "tutorial" field.

You're welcome to leave the category as "full", since you do offer a "full" product.

**Comment by tylampella** [comment]
> Awesome, glad to hear that! In that case, could you:
> 
> 1. add a few words to the "note" field to say which products are which, and
> 2. if you have it, a link to information about which button the user has to click in the "tutorial" field.
> 
> You're welcome to leave the category as "full", since you do offer a "full" product.

ok... I have updated the 'note' field. Thank you!

File: _data/hosting_providers.json [hunk 3]
```diff
@@ -39,6 +39,16 @@
...
   },
+  {
+    "name": "DreamHost",
+    "link": "https://www.dreamhost.com/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "https://www.dreamhost.com/hosting/ssl-tls-certificates/",
+    "plan": "",
+    "reviewed": "2020.1.22",
+    "note": "SSL automatically installed on DreamPress. All other products have SSL available via one click and rewnews automatically."
```
Comment (ohemorange) [inline _data/hosting_providers.json]:
```suggestion
    "note": "SSL automatically installed on DreamPress. All other products have SSL available via one click and renews automatically."
```

**Comment by tylampella** [comment]
> One typo then let's get this in, thanks for your work!

Ok.. all set! Thanks again

**Comment by tylampella** [comment]
> One typo then let's get this in, thanks for your work!

I accepted the typo fix.. anything else I need to do?

**Pull request merged**